Stripe/STPPinManagementService.m

```diff
@@ -0,0 +1,70 @@
+//
+//  STPAPIClient+PinManagement.m
+//  Stripe
+//
+//  Created by Arnaud Cavailhez on 4/29/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+#import "STPPinManagementService.h"
+#import "STPAPIRequest.h"
+#import "STPIssuingCardPin.h"
+#import "STPEphemeralKeyManager.h"
+#import "STPAPIClient+Private.h"
+
+@interface STPPinManagementService()
+@property (nonatomic, strong) STPEphemeralKeyManager *keyManager;
+@end
+
+@implementation STPPinManagementService
+
+- (instancetype)initWithKeyProvider:(id<STPIssuingCardEphemeralKeyProvider>)keyProvider {
+    self = [super init];
+    if (self) {
+        _keyManager = [[STPEphemeralKeyManager alloc] initWithKeyProvider:keyProvider apiVersion:[STPAPIClient apiVersion] performsEagerFetching:NO];
+    }
+    return self;
+}
+
+- (void)retrievePin:(__unused NSString *) cardId
+         verificationId:(__unused NSString *) verificationId
+        oneTimeCode:(__unused NSString *) oneTimeCode
+         completion:(__unused STPPinCompletionBlock) completion{
```

Comment:
these are used

Reply:
ah yes, they're used within a block though and XCode got confused, do you know how to resolve that?

Reply:
I just tried checking out the branch to investigate this and `STPPinManagementService.h` is missing

Reply:
oh, hm let me check on that!

Reply:
oh I had forgotten to add them, done now,
Does it change the `__unused` situation?

Reply:
style-nit: no space between the type and argument name. `completion` should also not have the `__unused` attribute...do you get an error when it's removed?